src/Charcoal/Source/DatabaseSource.php

```diff
@@ -643,7 +649,8 @@ public function updateItem(StorableInterface $item, array $properties = null)
             $key = $field->ident();
             if (in_array($key, $struct)) {
                 if ($key !== $model->key()) {
-                    $param = ':'.$key;
+                    $parseBindingCallback = $field->parseBinding();
+                    $param = $parseBindingCallback(':'.$key);
```

Comment:
Why does `parseBinding()` return a callback as opposed to just parsing the binding directly?

```suggestion
                    $param = $field->parseBinding(':'.$key);
```

Reply:
> I'm using a callback because it needs to be parsed in `DatabaseSource`
> `StorablePropertyTrait` sets the callback on the `PropertyField`, but it's only in `DatabaseSource` that the callback needs to be called with the parameter.
— @JoelAlphonso 

So, actually, the parsing of the binding is a callback to allow model properties (instances of `StorablePropertyInterface`) to customize the named placeholder. I presume, this is the equivalent of the "select expression" in order to convert back to geo spatial data.

Reply:
Indeed it is.

Reply:
Should this also be applied to `saveItem()`?

Reply:
Good catch. I fixed it in the latest commit